manifests/init.pp

```diff
@@ -71,6 +71,10 @@
     fail('This module only works on Debian or derivatives like Ubuntu')
   }
 
+  if $::apt_gpgversion and versioncmp ('2.0.26', $::apt_gpgversion) > 0 {
+    warning("You are running an old version of gpg. version=$::apt_gpgversion, please upgrade to at least 2.0.26")
```

Comment:
Yeah, I think I agree with @daenney here over the original ticket submitter. Sorry for not catching this sooner @tphoney